search-provider/eks-discovery-feed-provider.c

```diff
@@ -204,6 +205,27 @@ article_card_descriptions_cb (GObject *source,
   g_application_release (g_application_get_default ());
 
   GError *error = NULL;
+  EkncDomain *domain;
+  if (!(domain = eknc_engine_get_domain_for_app (engine, state->provider->application_id, &error)))
+    {
+      g_dbus_method_invocation_return_gerror (state->invocation, error);
```

Comment:
You will need to free the error after this, or use `g_dbus_method_invocation_take_error()` instead.